korepetycje-front/src/components/NavBar.vue

```diff
@@ -31,12 +31,19 @@ export default {
   .nav-links {
     display: flex;
     justify-content: flex-end;
-
     &__link {
-      padding: 20px 40px;
+      margin: 10px 0px;
+      padding: 10px 40px;
       font-size: 17px;
       text-decoration: none;
       color: #fff;
+      &.log {
```

Comment:
Jak zaczynasz nowy style to dodaj enter

Reply:
Dzięki za wskazówki, poprawie przy następnej okazji :)

Reply:
Ok, super :) To skoro pamiętasz to merguję